packages/docs/docs/guides/7-request storage subgraph/0-entities-storage.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Storage Subgraph Entities
+keywords: [Subgraph, Entities, Schema]
+description: Subgraph entities and descriptions.
+---
+
+# Transaction
+
+Description: get specific details of the transaction
+
+| Field                | Type      | Description                                                                                  |
+| -------------------- | --------- | -------------------------------------------------------------------------------------------- |
+| id                   | ID!       | a unique ID based on IPFS hash & position of the transaction in the document                 |
+| hash                 | String!   | IPFS hash                                                                                    |
+| channelId            | String!   | hash of the first transaction in the channel                                                 |
+| data                 | String    | transaction data, used for clear channels                                                    |
+| encryptedData        | String    | transaction data, used for encrypted channels                                                |
+| encryptionMethod     | String    | Encryption methods for channel key and transactions, concatenated together                   |
```

Comment:
The `encryptionMethod` looks like `ecies-aes-gcm`.  It's a concatenation of the asymmetric encryption scheme and the symmetric encryption scheme.

Reply:
hum, OK. I find it weird to phrase it like that, I guess we should link to the encryption specs. Because `ecies-aes-gcm` is one of the encryption methods we support. In RN context, it's 1 thing, not a concatenation...